components/layout/Navbar.tsx

```diff
@@ -140,28 +131,27 @@ export function Navbar({ }) {
             </a>
           </Link>
           <div className="w-full flex justify-end items-center gap-4">
+            <Button color="white" size="small" onClick={openModal}>
+              <MdEmail className="mr-2"/>
+              Subscribe
+            </Button>
             <LinkButton
-              link="https://app.tina.io/signin"
-              color="white"
-              size="small"
-            >
-              Log In
-            </LinkButton>
-            <LinkButton
-              link="https://app.tina.io/register"
+              link="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
```

Comment:
what happened to `https://app.tina.io/register`?
Does it still work?

Would definitely be nicer if `https://app.tina.io/register` redirected to the ugly URL

Reply:
Two completely different links